docs/services/google_analytics_js.rst

```diff
@@ -35,9 +38,11 @@ template.  Insert the tag at the bottom of the HTML head::
     <html>
     <head>
     ...
-    {% google_analytics_js %}
     </head>
+    <body>
     ...
+    {% google_analytics_js %}
+    </body>
```

Comment:
Same here: Adding a `<script>` tag to the HTML `<head>` is illegal. I guess the setup should be like above to be valid.

@mbourqui, @jcassee: Can you confirm?

Reply:
Similarly for analytics.js, according to the [google doc](https://developers.google.com/analytics/devguides/collection/analyticsjs#the_google_analytics_tag), it must be in that `<head>` tag:

> near the top of the \<head\> tag and before any other script or CSS tags

Reply:
That really scares me: Running this sample code through the W3 validator service tells it's valid HTML.

```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <script>
      alert("Valid HTML with JavaScript in the HEAD tag!");
    </script>
    <title>Foo</title>
  </head>
  <body>
  </body>
</html>
```

How could Google, as a consortium member, drag the standards so low? :astonished: